homework/index.js

```diff
@@ -1,47 +1,136 @@
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+function listContributors(contributors, parent) {
+  createAndAppend('h3', parent, { class: ' contributor - header', text: 'contributions' });
+  const ul = createAndAppend('ul', parent, { class: 'contributor-list' });
+  contributors.forEach(contributor => {
+    const itemOfContributor = createAndAppend('li', ul, { class: 'contributor-item' });
+    const divInItem = createAndAppend('div', itemOfContributor, {
+      class: 'contributor-container',
+    });
+    const contributorData = createAndAppend('div', itemOfContributor, { class: 'contributor-data' })
+    createAndAppend('img', divInItem, {
+      src: contributor.avatar_url,
+      class: 'contributor-avatar',
+    });
+    createAndAppend('a', contributorData, {
+      text: contributor.login,
+      href: contributor.html_url,
+      target: '_blank',
     });
+    createAndAppend('div', contributorData, {
+      text: contributor.contributions,
+      class: 'contributor-badge',
+    });
+  });
+}
+
+async function fetchRepositoryContributors(url, parent) {
+  try {
+    let info = await fetchJSON(url);
+    let repo = listContributors(info, parent);
+    return repo;
   }
+  catch (err) {
+    return renderError(err.name);
+  }
+}
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+function renderTableAndContributorsInfo(repositories, parent) {
+  const infoDiv = createAndAppend('div', parent, { class: 'tableInfo' });
+  const flexContainer = createAndAppend('div', parent, { class: 'flexContainer' })
+  const repoInfo = createAndAppend('div', flexContainer, { class: 'left-div whiteframe' });
+  const mainDiv = createAndAppend('div', repoInfo, { id: 'container' });
+  const contributorDiv = createAndAppend('div', flexContainer, { class: 'right-div whiteframe' });
+  createAndAppend('p', infoDiv, { text: 'HYF REPO' });
+  const select = createAndAppend('select', infoDiv, { class: 'repo-selector' });
+  repositories.sort((a, b) => a.name.localeCompare(b.name, 'en', { sensitivity: 'base' }));
+  repositories.forEach((repository, index) => {
+    createAndAppend('option', select, { text: repository.name, value: index });
+  });
+  renderTableInfo(mainDiv, repositories[0]);
+  fetchRepositoryContributors(repositories[0].contributors_url, contributorDiv);
+  select.addEventListener('change', event => {
+    mainDiv.innerHTML = '';
+    contributorDiv.innerHTML = '';
+    renderTableInfo(mainDiv, repositories[event.target.value]);
+    fetchRepositoryContributors(
+      repositories[event.target.value].contributors_url,
+      contributorDiv,
+    );
+  });
+}
 
-  window.onload = () => main(HYF_REPOS_URL);
+async function main(url) {
+  const root = document.getElementById('root');
+  try {
+    let data = await fetchJSON(url);
+    let render = renderTableAndContributorsInfo(data, root);
```

Comment:
These two `let` statements should be `const`. You should be getting an ESLint warning about that and a red underline in VSCode. Because the ESLint check performed when you submit your PR automatically fixes these style errors, you code still passes the test. However, you should strive to make you code correct without it needing to be fixed automatically by ESLint.